_rules/line-height-not-important-78fd32.md

```diff
@@ -21,20 +21,22 @@ acknowledgments:
 
 ## Applicability
 
-This rule applies to any HTML element with multi-line text that is [visible][] and for which the `style` attribute [declares][declared] the [line-height][] CSS property.
+This rule applies to any HTML element with [visible][] text that includes a [line-break](https://www.w3.org/TR/css-text-3/#line-breaking) and for which the `style` attribute [declares][declared] the [line-height][] CSS property.
```

Comment:
I'm a bit unsure about the "line break" (CSS) definition:
> When inline-level content is laid out into lines, it is broken across line boxes. Such a break is called a line break. When a line is broken due to explicit line-breaking controls (such as a preserved newline character), or due to the start or end of a block, it is a forced line break. When a line is broken due to content wrapping (i.e. when the UA creates unforced line breaks in order to fit the content within the measure), it is a soft wrap break. The process of breaking inline-level content into lines is called line breaking.

On one hand, it seems to only concern inline-level content, but on the other hand it mentions start/end of block as being one kind of line breaks. That puzzles me a bit 🤔 🧩 

Where it gets is to the following:
```html
<div style="line-height: 1px !important">
  <p>Hello</p>
  <p>World</p>
</div>
```
Does the `div` contain a line break or not? On one hand it has no (direct) inline content, but on the other hand the start/end of `p` is a start/end of block, hence a "force line break". Point is that this is not failing the SC (due do the paragraph spacing), and therefore shouldn't fail the rule; but if this contains a line break, then it fails the rule.

---
Maybe just using
```suggestion
This rule applies to any HTML element with [visible][] text that includes a [soft wrap break](https://www.w3.org/TR/css-text-3/#soft-wrap-break) and for which the `style` attribute [declares][declared] the [line-height][] CSS property.
```
will at least make sure we restrict the rule to line breaks created by UA for display reason, (not considering author line break). This is maybe not enough, but should ensure we don't have false positives.

Reply:
Understood, I was initially intending the example that you had above not to be applicable, but given the forced line break you pointed how I see how it could cause problems. I am fine going using soft-wrap as well.